evalai/utils/challenges.py

```diff
@@ -174,3 +174,52 @@ def display_participated_or_hosted_challenges(is_host=False, is_participant=Fals
                 pretty_print_challenge_data(challenge)
         else:
             echo("Sorry, no challenges found!")
+
+
+def pretty_print_challenge_phase_split_data(splits):
+    """
+    Function to print the details of a Challenge Phase Splits.
+    """
+    for split in splits:
+        br = style("----------------------------------------"
```

Comment:
I just noticed this, but instead of typing out so many hyphens, you can do something like `'-'*N` where N is the number of times you want to print it.

Reply:
I'll start an issue to solve this! Thank you.